src/main/java/games/strategy/engine/framework/startup/ui/MetaSetupPanel.java

```diff
@@ -95,6 +100,24 @@ private void layoutComponents() {
     // top space
     add(new JPanel(), new GridBagConstraints(0, 100, 1, 1, 1, 1, GridBagConstraints.CENTER, GridBagConstraints.BOTH,
         new Insets(00, 0, 0, 0), 0, 0));
+
+    if (ClientSetting.SHOW_BETA_FEATURES.booleanValue()) {
+      final JButton mapCreator =
+          SwingComponents.newJButton("Run the Map Creator", e -> ProcessRunnerUtil.runClass(MapCreator.class));
+      add(mapCreator, new GridBagConstraints(0, 10, 1, 1, 0, 0, GridBagConstraints.CENTER, GridBagConstraints.NONE,
+          new Insets(10, 0, 0, 0), 0, 0));
+
+      final JButton m_mapXmlCreator = SwingComponents
```

Comment:
I know this was just copy-pasted from the engine preferences dialog, but might as well remove the `m_` prefix since this line was changed.